### ISSUE TITLE
When configured to use User namespace, links to actionees does not work

### ISSUE THREAD

**Issue by darenwelsh** [enterprisemediawiki/SemanticActions#14]
![image](https://user-images.githubusercontent.com/6728676/62892856-d9a26080-bd0e-11e9-857a-4a2739b58adb.png)

**Comment by darenwelsh** [comment]
I think only Template:Actionable needs to be modified. This means there will be two different versions depending on the config variable.

**Comment by darenwelsh** [comment]
```
ASSIGNED_TO

--><tr><td>'''Actionees'''</td><td>{{#if:{{{ASSIGNED_TO|}}}|{{#arraymap:{{{ASSIGNED_TO|}}}|,|x|[[User:x|x]]|,}}</td></tr>|}}<!--
```

**Comment by darenwelsh** [comment]
With this method, the links to usernames on a standard SMW query results table (like one used on a user page) will still be missing the "User:" prefix.

Either the query results for table format will require a template output (not very nice to the wiki users) or the properties being set need to include the namespace prefix. I think the latter is the better way to go.

**Comment by darenwelsh** [comment]
This needs to also be fixed in 
Template:Actionable board row

**Comment by darenwelsh** [comment]
something other than `x` should be used in the arraymap (for both config options)

**Comment by darenwelsh** [comment]
It might be better to modify

```
-->{{#set:
|Action ID={{#sub:{{FULLPAGENAME}}|7|0}} 
|Summary={{{SUMMARY|}}}
|Assigned to={{{ASSIGNED_TO|}}}|+sep=,
```

to add "User:"

Note that other pages will need to be modified so "User:" isn't displayed in the action block links.

**Comment by darenwelsh** [comment]
https://mwstake.org/mwstake/index.php?title=Template%3AActionable&type=revision&diff=1810&oldid=1777

and

https://mwstake.org/mwstake/index.php?title=Template%3AActionable_board_row&type=revision&diff=1809&oldid=1775

**Comment by darenwelsh** [comment]
Fixed by #17

**Comment by darenwelsh** [comment]
Had to add 7c25a5a5cfd7d9ffd118ff19cb599a4a38f10926
